doc/src/sgml/xsltproc_depfile_wrapper.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+import subprocess
+import argparse
+from unittest import result
+
+parser = argparse.ArgumentParser(
+    description='generate dependency file for docs')
+
+parser.add_argument('--targetname', type=str, required=False, nargs='+')
+parser.add_argument('--depfile', type=str, required=False)
+parser.add_argument('--xsltproc', type=str, required=True)
+
+parser.add_argument('xsltproc_flags', nargs='*')
+
+args = parser.parse_args()
+
+if args.depfile:
+    depfile_result_list = []
+
+    command = [args.xsltproc, '--load-trace'] + args.xsltproc_flags
+    # --load-trace flag displays all the documents
+    # loaded during the processing to stderr
+    res = subprocess.run(command, stderr=subprocess.PIPE, check=True)
+    # if exit code is different than 0, exit
+    if res.returncode:
+        exit(res.returncode)
+
+    # make dependencies list
+    dependencies = res.stderr.decode("utf-8").split('\n')
+
```

Comment:
Hm. Can we rely on this actually being utf-8?

Reply:
You are right, deleted that part.